stglib/tests/data/Bang_--_MBW.csv

```diff
@@ -0,0 +1,12383 @@
+﻿"Plot Title: Bang -- MBW"
```

Comment:
Can you set the csv files to be in git lfs please?

Reply:
yes. sorry I thought they were.